packages/@aws-cdk/aws-synthetics-alpha/lib/canary.ts

```diff
@@ -195,9 +200,17 @@ export interface CanaryProps {
    * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-synthetics-canary.html#cfn-synthetics-canary-deletelambdaresourcesoncanarydeletion
    *
    * @default false
+   * @deprecated use autoDeleteLambda instead
```

Comment:
Maybe explain why this is in the property body. Something like "this feature has been deprecated by the service team; instead, use `autoDeleteLambda` which will use a Custom Resource to achieve the same effect"

?